frontend/src/v5/ui/components/authTemplate/authTemplate.styles.ts

```diff
@@ -75,6 +75,8 @@ export const BackgroundOverlay = styled(LoginBackground)`
 	left: 0;
 	right: 0;
 	overflow: hidden;
+	object-fit: cover;
+	object-position: 0 0;
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/62481250/253064a7-c992-4d07-87ff-b7aeddd01232)
On wide screens the background doesn't extend all the way across. I think changing max-width to be min-width and perhaps changing object-fit to "fill" fixes this.